### ISSUE TITLE
Documentation error for multiple word data types

### ISSUE THREAD

**Issue by jmellicker** [crate/crate#8605]
**CrateDB version**:
3.2.5

**Problem description**:

In the docs:
[https://crate.io/docs/crate/reference/en/latest/general/ddl/data-types.html](https://crate.io/docs/crate/reference/en/latest/general/ddl/data-types.html)

there is this example code for creating a table:

```sql
create table my_table13 (
    title text,
    author object as (
        name text,
        birthday timestamp with time zone
    )
)

```
which, when run in the Crate Console, results in this error:

```
Error!

SQLActionException[SQLParseException: line 5:28: no viable alternative at input 'author object as (\n name text,\n birthday timestamp with']

```

Changing the query to:

```sql
create table my_table13 (
  birthday timestamp with time zone
)
```

results in:

```
Error!

SQLActionException[SQLParseException: line 2:22: mismatched input 'with' expecting {',', ')', 'NOT', 'PRIMARY KEY', 'INDEX', 'STORAGE'}]
```

**Comment by kovrus** [comment]
Hello @jmellicker,

In the statements that you have posted you use data type names that have not been [released](https://crate.io/docs/crate/reference/en/latest/appendices/release-notes/unreleased.html#id1) yet. It will be available in the future CrateDB releases ( >= 4.0.0). Please, see the [documentation](https://crate.io/docs/crate/reference/en/3.2/general/ddl/data-types.html) for CrateDB data types that corresponds to the version you use.

However, you can use those statements in one of the CrateDB [nightly (https://crate.io/download/#cratedb) builds.

**Comment by jmellicker** [comment]
Ah! Thanks!
